packages/video_player/video_player_web/android/.gitignore

```diff
@@ -0,0 +1,8 @@
+*.iml
```

Comment:
nit: This can be deleted.

Reply:
I'm keeping the template list as the easy choice, FWIW someone may decide to open this project with Android Studio...

Reply:
I think our root .gitignore would still be applied here in that case, since AS should be deferring to git which would be able to tell that this is part of a larger repo. But either way it's fine to keep this if you'd prefer, it's not doing any harm and matches the template.